lib/sanitize_ext/sanitize_config.rb

```diff
@@ -136,7 +136,7 @@ module Config
       attributes: {
         'audio' => %w(controls),
         'embed' => %w(height src type width),
-        'iframe' => %w(allowfullscreen frameborder height scrolling src width),
+        'iframe' => %w(allowfullscreen height scrolling src style width),
```

Comment:
I think allowing `style` might actually introduce potential security issues. Not sure this is necessary.

Reply:
Should I just leave `frameborder` as allowed here, for any external embeds that might still use it?

Reply:
Allowing `style` attributes *should* be safe, although there have been vulnerabilities in our sanitizing library in the past, and limiting the surface area should be best.

And if we do decide to allow `style` attributes, it probably makes sense to only allow specific properties like `border`. And keep support for `frameborder` as well.

Reply:
Thanks! To keep this simple, I've reverted this file, and kept the changes focused to the places where Mastodon was authoring the `frameborder` attribute

Reply:
Afaik the `html` bit you modified still goes through sanitization, so by using `style` instead, it will get sanitized away and won't work.

Reply:
Good call. I added back `style`, but left `frameborder` for any legacy content that might need it for the same affect